src/garage/torch/policies/gaussian_mlp_policy.py

```diff
@@ -36,6 +36,8 @@ def forward(self, inputs):
     def get_action(self, observation):
         """Get a single action given an observation."""
         with torch.no_grad():
+            if(type(observation) != torch.Tensor):
```

Comment:
the type of the observation is either torch.tensor or it's np.ndarray, it's not both.